app/components/partner/client-plan/template.hbs

```diff
@@ -3,32 +3,32 @@
     {{#if this.getClientPlan.isRunning}}
       {{fa-icon 'circle-o-notch fa-spin text-lightgray'}}
     {{else if this.clientPlan}}
-      <div local-class='plan' data-test-payment-plan data-test='client-plan'>
+      <div local-class='plan' data-test-payment-plan data-test-client-plan>
```

Comment:
`data-test-payment-plan` can be removed right?

Reply:
yes, I'll remove this and check is there any dependency check for this attr